doc/rst/developer/bestPractices/buildingdocs.rst

```diff
@@ -1,15 +1,230 @@
 =============================
+The live Chapel web documentation lives at https://chapel-lang.org/docs/.
+All of the source for the documentation is in the chapel source repository, and
+this note talks about how those are organized and built into the html documentation
+seen on the website.
 
 The easiest way to build the Chapel web documentation is to use the top level
-make target::
+of your chapel github clone
 
+make target::
    [g]make docs
 
 This will build the documentation for the standard and internal modules using
 chpldoc, and symlink all of the ``$CHPL_HOME/doc/`` documents over to
 a sphinx project that lives in ``$CHPL_HOME/doc/``. Once built,
 the documentation can be viewed at ``$CHPL_HOME/doc/html/index.html``.
+
+The html documentation is also generated by the github CI (continuous integration)
+process.  You can go to your pull request in github, click on "Checks",
+then "Ci" (on the left), and then scroll down to see the "documentation"
+(at the very bottom).  It enables you to download a zip file of the built
+documentation.
+
+Recent and previous releases of the documentation can be found at
+https://chapel-lang.org/docs/.
+
+
+How the HTML is Generated
+=========================
+
+The Chapel documentation shown at https://chapel-lang.org/docs/ is generated
+in three different ways:
+
+#.  Restructure Text files (with a .rst extension) are converted into .html
+    files via sphinx (https://sublime-and-sphinx-guide.readthedocs.io/).
+#.  Comments and code from Chapel source files are converted into .rst files
+    using ``chapel/doc/util/chpl2rst.py``.
+#.  Some comments and code from Chapel source files are converted into .rst
+    files using ``chapel/tools/chpldoc/``, which will leave out TODOs,
+    FIXMEs, and other parts of the comments.
+
+========================================
+Contributing to the Chapel Documentation
+========================================
+
+See https://chapel-lang.org/contributing.html for general instructions on how
+to contribute to Chapel.  Contributing to the documentation is similar except
+you will want to build the documentation (see above) and check the generated
+html files before creating a pull request.  Also note that if you are editing
+.chpl files in the primers directory for example, you will need to 
+ref:develop-and-test-contributions-locally.
+
+
+Editing the Source .rst and .chpl files
+=======================================
+
+The .chpl files are text files that can be edited with any text editor.
+See ``$CHPL_HOME/highlight/README.md`` for information about some available 
+extensions for editors to highlight Chapel code.
+
+The .rst files are also text.  The Chapel files that are being converted into .rst
+files have rst text in comments.  There are some previewers for .rst text in 
+vscode and atom, but none of them are quite as useful as editing a .rst file
+in github and using the preview option.  The way to see all of the links that
```

Comment:
GitHub again, but Chapel is capitalized all through here, which seems to indicate it should be above as well.

Reply:
I decided to capitalize Github throughout.  Good suggestion!